src/App.tsx

```diff
@@ -1,24 +1,162 @@
-/* eslint-disable max-len */
+/* eslint-disable react-hooks/rules-of-hooks */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, {
+  useMemo,
+  useEffect,
+  useState,
+  useCallback,
+} from 'react';
 import { UserWarning } from './UserWarning';
+import { Todo } from './types/Todo';
+import { createTodo, deleteTodo, getTodos } from './api/todos';
+import { TodoList } from './components/TodoList';
+import { TodoForm } from './components/TodoForm';
+import { Error } from './components/Error';
+import { Filters } from './types/Filters';
+import { Filter } from './components/Filter';
 
-const USER_ID = 0;
+const USER_ID = 10892;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [error, setError] = useState<string | null>(null);
+  const [filter, setFilter] = useState(Filters.ALL);
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+  const [loadingTodo, setLoadingTodo] = useState([0]);
```

Comment:
That's not a suggestion, only a question. Why did you decide to store the ids of loading todos? During student review, I've seen the other person also do it, but can't wrap my head around why))

Reply:
nvm, I'm doing today's task and now I GET IT 🤦

Reply:
I also tried to do it without loadindgTodo state and didn't understand why it works in a wrong way😁